### PR TITLE
Add latest STM32H7 and STM32L4 boards

### PR DESCRIPTION
**Pull request by jeromecoutant**
### Description

Add latest STM32H7 and STM32L4 boards

### Pull request type

    [ ] Fix
    [ ] Refactor
    [x] Target update
    [ ] Functionality change
    [ ] Docs update
    [ ] Test update
    [ ] Breaking change

### Reviewers

@MarceloSalazar

### CODE REVIEW

File: src/mbed_os_tools/detect/platform_database.py [hunk 1]
```diff
@@ -178,10 +179,15 @@
...
         u"0855": u"DISCO_L562QE",
+        u"0858": u"DISCO_H750B",
+        u"0859": u"DISCO_H7B3I",
         u"0860": u"NUCLEO_H7A3ZI_Q",
+        u"0863": u"DISCO_L4P5G",
+        u"0865": u"NUCLEO_L4P5ZG",
         u"0866": u"NUCLEO_WL55JC",
         u"0871": u"NUCLEO_H723ZG",
         u"0872": u"NUCLEO_G0B1RE",
+        u"0875": u"DISCO_H735ZG",
```
Comment (MarceloSalazar) [inline src/mbed_os_tools/detect/platform_database.py:190]: Proposal to match online database:
```suggestion
        u"0875": u"DISCO_H735G",
```

Comment (jeromecoutant) [reply src/mbed_os_tools/detect/platform_database.py]: Corrected

**Comment by MarceloSalazar** [comment]
Board IDs and target names match online database.
Ready to get in.

**Comment by jeromecoutant** [comment]
Please release v1.8.12 after merge
Thx

**Comment by MarceloSalazar** [comment]
@Patater can you please review and merge this one?

**Review by Patater — APPROVED** [review]

**Comment by jeromecoutant** [comment]
Thx!
Is it possible to release v1.8.12 ?

**Pull request merged**